hack/update_bundle_catalog.sh

```diff
@@ -7,64 +7,102 @@
 
 set -euo pipefail
 
+: ${OPM:=$(which opm)}
+echo "using opm from ${OPM}"
 # check if opm version is v1.39.0 or exit
-if ! opm version | grep -q "v1.39.0"; then
+if ! ${OPM} version | grep -q "v1.39.0"; then
   echo "opm version v1.39.0 is required"
   exit 1
 fi
 
-TAG="latest"  # Set the operator version
-BUNDLE_TAG="0.0.1"    # Set the bundle version
+: ${YQ:=$(which yq)}
+echo "using yq from ${YQ}"
+# check if yq exists
+if [ -z "${YQ}" ]; then
```

Comment:
`command -v yq` will check if it's really executable, might be more precise test